examples/Tungstun-train-set/README.md

```diff
@@ -3,16 +3,16 @@ Szlachta, Bartók & Csányi Accuracy and transferability of Gaussian
 approximation potential models for tungsten PRB 2014
 
 * step 0
-frame_select.py -fxyz Tungstun_GAP_6.xyz --n 1000 --prefix subset --algo random
+python ../../scripts/frame_select.py -fxyz Tungstun_GAP_6.xyz --n 1000 --prefix subset --algo random
```

Comment:
We decided to just use the name of the script, instead of the relative path in the examples,
because we expect people to install it